README.md

```diff
@@ -1,4 +1,4 @@
-# Terraform Provider for `ko`
+# Terraform Provider for [google/ko](https://github.com/google/ko)
```

Comment:
```suggestion
# Terraform Provider for [`ko`](https://github.com/google/ko)
```

We're moving `ko` soon, so referring to it as "google/ko" will probably just require another update in the future. The link should redirect after the move at least.